sqldelight-compiler/src/test/kotlin/com/squareup/sqldelight/core/queries/MutatorQueryTypeTest.kt

```diff
@@ -351,6 +357,243 @@ class MutatorQueryTypeTest {
       |""".trimMargin())
   }
 
+  @Test fun `types bind fine in HSQL`(dialect: DialectPreset) {
```

Comment:
Sorry for how verbose all these tests are 😭

Reply:
its np, i mostly find them important for tracking regressions than reviewing for codegen